xds/internal/balancer/clusterresolver/e2e_test/aggregate_cluster_test.go

```diff
@@ -18,16 +18,18 @@ package e2e_test
 
```

Comment:
Are these changes orthogonal to your other testing PR?

Reply:
Yes. I basically found this issue because of the test changes that I'm making in that PR, i.e changing the fake DNS resolver with a real one which *can* call back inline with updates/errors. So, I will wait for this PR to be merged before merging the test PR.

Reply:
Alright sounds good.